ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -252,13 +289,16 @@ TensorFilterLlamaCpp::invoke_dynamic (GstTensorFilterProperties *prop,
 
   for (auto id : prompt_tokens) {
     char buf[128] = { 0 };
+    GstTensorMemory *output = nullptr;
 
     n = llama_token_to_piece (vocab, id, buf, sizeof (buf), 0, true);
     if (n < 0) {
       throw std::invalid_argument ("Failed to convert token to piece");
     }
-    std::string s (buf, n);
-    result += s;
+    output = g_new0 (GstTensorMemory, 1);
+    createOutputTensor (buf, n, output, prop);
```

Comment:
If `nnstreamer_filter_dispatch_output_async` is always called after `createOutputTensor`, how about call this function inside `createOutputTensor`?

Reply:
Adding a dispatch function to createOutputTensor reduces clarity as a single function now performs two roles. It also becomes difficult to make the function name reflect its original role, so I thought it would be structurally preferable to separate tensor creation and dispatch processing by function.